Makefile

```diff
@@ -18,8 +18,9 @@ UP_BIN=$(BIN_DIR)/up
 MEMCACHED_BIN=$(BIN_DIR)/memched
 PROMETHEUS_BIN=$(BIN_DIR)/prometheus
 GOJSONTOYAML_BIN=$(BIN_DIR)/gojsontoyaml
+JSONNET?=$(BIN_DIR)/jsonnet
 # We need jsonnet on CI; here we default to the user's installed jsonnet binary; if nothing is installed, then install go-jsonnet.
-JSONNET_BIN=$(if $(shell which jsonnet 2>/dev/null),$(shell which jsonnet 2>/dev/null),$(BIN_DIR)/jsonnet)
+JSONNET_BIN=$(if $(shell which jsonnet 2>/dev/null),$(shell which jsonnet 2>/dev/null),$(JSONNET))
```

Comment:
to have this be a bit more consistent, let's keep the other one as $(JSONNET_BIN) make this special variable something explicit like $(JSONNET_LOCAL_OR_INSTALLED)